app/views/items/show.html.erb

```diff
@@ -85,7 +85,7 @@
         font-variation-settings: 'FILL' 1
     }
 
-    #notFilledEditItemIcon {
+    #ownerFunctionIcons {
```

Comment:
ownerFunctionIcons should be a class, not an id if used multiple times
```suggestion
   .ownerFunctionIcons {
```
This would have to be changed in lines 130-135 as well